go

```diff
@@ -34,7 +34,7 @@ def_command :update_gems, 'Update Ruby gems' do |gems = []|
 end
 
 def_command :test, 'Execute automated tests' do |args = []|
-  exec_cmd "bundle exec rake test #{args.join ' '}"
```

Comment:
@mbland I tried changing this line back like you suggested, and it didn't work. I found on the web that it probably needed `s/test/spec`, but that still isn't running the specs. Not sure where to go from there.